gulpfile.babel.js

```diff
@@ -135,6 +138,75 @@ gulp.task('core-test', ['prepare', 'core', 'core-dts-test'], (done) => {
   ).start();
 });
 
+gulp.task('core-test-separately', ['prepare', 'core', 'core-dts-test'], (done) => {
+  // Usage:
+  //    # run all unit tests
+  //    gulp core core-test-separately
+  //
+  //    # run only specified unit tests
+  //    gulp core core-test-separately --target core/src/elements/ons-navigator/index.spec.js
+
+  (async () => {
+    let listOfSpecFiles;
+    if ($.util.env.target) { // if --target option is specified
+      listOfSpecFiles = [ path.join(__dirname, $.util.env.target) ];
+    } else {
+      listOfSpecFiles = glob.sync(path.join(__dirname, 'core/src/**/*.spec.js'));
+    }
+
+    // Separately launch Karma server for each spec file
+    try {
+      for (let i = 0 ; i < listOfSpecFiles.length ; i++) {
+        $.util.log($.util.colors.blue(path.relative(__dirname, listOfSpecFiles[i])));
+
+        // Pass parameters to Karma config file via process.env
+        process.env.SPEC_FILES_SPECIFIED = true;
```

Comment:
@argelius 
I found that we can use `global` instead of `process.env` and it also worked.
I have fixed `gulpfile.babel.js` and `karma.conf.js` to use `global`.